app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/email/EmailController.scala

```diff
@@ -186,7 +188,8 @@ trait EmailController[Journey <: JourneyStatus, VerificationCompleteJourney <: J
                   _ <- EitherT[Future, Error, Unit](updateSession(sessionStore, request) { s =>
                         s.copy(
                           journeyStatus     = Some(journey),
-                          emailToBeVerified = Some(emailToBeVerified.copy(verified = true))
+                          emailToBeVerified = Some(emailToBeVerified.copy(verified = true)),
+                          subscriptionDetailUpdated = if(updateSubscriptionDetailUpdated) Some(SubscriptionDetail.Email) else None
```

Comment:
`updateSubscriptionDetailUpdated` - what does this mean? it's been updated twice?

Reply:
it was meant to mean "update the subscriptionDetailsUpated field in the session data". Wasn't quite sure what a nice name would be for that. Maybe `updateSessionSubscriptionDetailUpated`?

Reply:
what about `updateSubscriptionDetailChangeFlag`? bit of a mouth full but it's clear I think

Reply:
yeah, I think that sounds better. I'll change it to that (with `change` in the past tense though I think)

Reply:
I renamed it now